src/iop/denoiseprofile.c

```diff
@@ -701,8 +701,8 @@ void tiling_callback(struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t
 
   if(d->mode == MODE_NLMEANS || d->mode == MODE_NLMEANS_AUTO)
   {
-    const int P = ceilf(d->radius * fminf(roi_in->scale, 2.0f) / fmaxf(piece->iscale, 1.0f)); // pixel filter size
```

Comment:
not sure about the inner fmax and fmin. Aren't they redundant ?

Reply:
honestly I don't know at all

Reply:
ok, let's merge that. Thanks !